flow-server/src/main/java/com/vaadin/flow/server/communication/PwaHandler.java

```diff
@@ -81,6 +84,9 @@ private void init(PwaRegistry pwaRegistry) {
                         }
                         try (OutputStream out = response.getOutputStream()) {
                             icon.write(out);
+                        } catch (UncheckedIOException ex) {
+                            LoggerFactory.getLogger(PwaHandler.class)
+                                    .debug("Error serving PWA image", ex);
```

Comment:
Maybe?
```suggestion
                                    .debug("Error serving PWA icon", ex);
```
just to be more consistant with the namings.

Reply:
done

Reply:
Thank you vm! 😊